runelite-client/src/main/java/net/runelite/client/ui/overlay/tooltip/TooltipOverlay.java

```diff
@@ -122,6 +130,11 @@ private Dimension renderTooltips(Graphics2D graphics, List<Tooltip> tooltips)
 				mousePosition.move(mouseCanvasPosition.getX(), mouseCanvasPosition.getY() + offset + newBounds.height);
 			}
 
+			if (tooltips.size() > 1 && this.runeLiteConfig.tooltipPosition() == TooltipPositionType.ABOVE_CURSOR)
+			{
+				mousePosition.translate(CURSOR_OFFSET, 0);
```

Comment:
After testing this ingame, I can say confidently this is not the right behavior. We should not be translating tooltips to the right of the cursor when there are multiple present--they should instead tile upwards like so:

```
--------------
|  tooltip 1 |
--------------
---------------------
| multiline tooltip |
| another line      |
| foobar            |
---------------------
|\
|  \
| ___\
|/ \\
    \\
```

Reply:
Alright that sounds better to me. I appreciate the art you made as well 😅

Reply:
![image](https://user-images.githubusercontent.com/13258516/75592537-8b597280-5a50-11ea-8daf-6ee584473060.png)